Add the `createTestingToken()` backend SDK docs (#947)
clerk/clerk-docs: docs/references/backend/overview.mdx

- [`deletePhoneNumber()`](/docs/references/backend/phone-numbers/delete-phone-number)

## Testing tokens operations

A testing token can be used to bypass the Bot Protection rules on E2E testing.

> Testing tokens operations are exposed under `clerkClient.testingTokens`. The following methods are available:

Reviewer: ```suggestion
Testing tokens operations are exposed under `clerkClient.testingTokens` with the [`createTestingToken()`](/docs/references/backend/testing-tokens/create-testing-token) helper method.
```

Reviewer: Let's not use a bulleted list for this method since it's the only one

> - [`createTestingToken()`](/docs/references/backend/testing-tokens/create-testing-token)

Reviewer: ```suggestion
```
The above suggestion makes this line unnecessary

- [`deletePhoneNumber()`](/docs/references/backend/phone-numbers/delete-phone-number)

## Testing tokens operations

A testing token can be used to bypass the Bot Protection rules on E2E testing.

Testing tokens operations are exposed under `clerkClient.testingTokens`. The following methods are available:

> ### Testing tokens `create` operations

Reviewer: ```suggestion
```

I think we can just delete this heading and keep the bulleted list.

> - [`createTestingToken()`](/docs/references/backend/testing-tokens/create-testing-token)

Reviewer: Will we be adding more items to this list in this PR?

Author: @S3Prototype no we are not going to add anything else
